onnxruntime/test/providers/cpu/controlflow/loop_test.cc

```diff
@@ -108,44 +113,59 @@ static const ONNX_NAMESPACE::GraphProto CreateSubgraph(const RunOptions& options
        |             |                           |                          [Constant]
   iter_num_float  sum_0                          |          sum_0             /
        |           / | \                         |              \            /
-    [Concat]------/  |  \---------------------[Concat]           \--[Less]--/
-       |             |                           |                     |
-       |         [Identity]                      |                     |
+  (if scalar)     /  |  \---------------------[Concat]           \--[Less]--/
+  [Unsqueeze]    /   |                           |                     |
```

Comment:
Thank you for being thorough with the unit test updates, including the ascii art :)